devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Regarding the source for the Ansible script, there is a link in the trello and in an a previous email I sent. I've been testing the Nginx/Rhel setup.[nginx/rhel](https://github.com/ansible/ansible-examples)